priya/javascript/JsAssign6Q1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
Indentation Little bit
```suggestion
    <head>
```